release-notes/major12/12.0.0/migration.md

```diff
@@ -12,8 +12,11 @@ This page describes how to update Valtimo from the previous version to the curre
 
   Scope: back-end
 
-  `valtimo-dependencies` and `valtimo-gzac-depenencies` should no longer be used for `dependencyManagement` or `platform`.
+  1. `valtimo-dependencies` and `valtimo-gzac-dependencies` should no longer be used for `dependencyManagement` or `platform`.
   This functionality has been replaced by [valtimo-dependency-versions](../../../getting-started/modules/core/valtimo-dependency-versions.md).
+  2. If present, remove the dependency to `org.hibernate:hibernate-envers` 
+  3. If present, upgrade dependency `org.hibernate:hibernate-core` to `6.0.2.Final` or higher.
+  4. If present, upgrade dependency `com.ritense.valtimo:valtimo-testcontainers` to `0.0.2-SNAPSHOT` or higher.
```

Comment:
The dependency isn't public so shouldn't be mentioned here imo

Reply:
I've thought about that. But having this in the migration notes, will prevent a lot of troubleshooting in our implementation teams .

Reply:
I see your point, but I think the downside of having this in the public docs is greater than the upside. Let's find out who uses this library and communicate directly to them.

Reply:
What are the downsides?

Reply:
Removed